docs/introduction.rst

```diff
@@ -0,0 +1,48 @@
+Introduction
+=============
+
+`Dataduct <https://github.com/coursera/dataduct>`__ is a wrapper built
+on top of `AWS
+Datapipeline <http://docs.aws.amazon.com/datapipeline/latest/DeveloperGuide/what-is-datapipeline.html>`__
+which makes it easy to create ETL jobs. All jobs can be specified as a
+series of steps in a YAML file and would automatically be translated
+into datapipeline with appropriate pipeline objects.
+
+Top features include visualizing pipeline dependencies, extract data
```

Comment:
The grammar in this paragraph needs to be changed.

extract -> extracting, Local -> local, run -> running, load -> loading, complete -> completing.

The final sentence is also not a sentence.